master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md

```diff
@@ -34,6 +36,21 @@ Once installed, you can try out NetworkPolicy by following the [simple policy gu
 
 Below are a few examples for how to get started.
 
+### Using Kubernetes' addon-manager
+
+The Kubernetes [addon-manager](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/addon-manager)
+can be used for deploying Calico alongside with other cluster addons. Monitors
+`/etc/kubernetes/addons` for manifests and ensures they
+are applied to the cluster. The following lines need to be run on the master of
```

Comment:
referring to them as lines is a bit weird. Can we do:

"Run the following commands on a kubernetes master which has the addon-manager installed:"